roadmap/2022.md

```diff
@@ -54,3 +54,30 @@ Misc:
   - [ ] Finish changes regarding the API keys
   - [ ] Finish changes regarding the new index routes
   - [ ] Add the tenant tokens methods
+
+## Q2
+
+
+Analytics:
+- [ ] Finish the implementation in the remaining SDKs (Java, Javascript, Rust)
+- [ ] Plan how we will implement the analytics on the integrations, tools & plugins.
+- [ ] _(may not be required)_ Define the specs to support analytics in some tools with core team.
+- [ ] Implement it on the integrations.
+- [ ] Implement it on the plugins.
+
+Processes:
+- [ ] Change the Integrations' Team developers scope.
+
+Improvements:
+- [ ] Add a helpful docker/compose setup in every integration.
+- [ ] Code coverage:
+  - [ ] Define the tool which will be used (codecov, coveralls, others).
+  - [ ] Implement it in the SDKs.
+  - [ ] Implement it on the integrations which makes sense to have coverage.
+
+Meilisearch releases:
+- [ ] v0.27 work
```

Comment:
Should we add v0.28 ?

Reply:
Yeah, I think, but currently, I don't have the time to dig into the planned changes for the v0.28.

Reply:
And I don't want to put "v0.28 a lot of things" 😅